CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [2.5.1](https://github.com/rewardops/rewardops-sdk-node/compare/v2.5.0...v2.5.1) (2021-07-08)
+
+### Refactor
+
+- unify log format with mx-apps ([#79](https://github.com/rewardops/rewardops-sdk-node/issues/79)) ([42b3403](https://github.com/rewardops/rewardops-sdk-node/commit/179754b8360d0dcf1932d6ec0c6223454135c5c9))
```

Comment:
Question: Does our MX log format conform with anything from the broader community?

Reply:
Hmm. I don't think so. It's only based on the request coming from the devOps.

I did some research online, [NCSA Common](https://publib.boulder.ibm.com/tividd/td/ITWSA/ITWSA_info45/en_US/HTML/guide/c-logs.html#common) seems to be more standard. But I think in the end it all comes down to preferences as long as the log is easy to read for both human and machine.